test/test_dependency_versions.py

```diff
@@ -27,7 +27,9 @@
 
         print('Gathered versions', versions)
 
-        for package_name in ['pip', 'setuptools', 'wheel']:
+        packages_to_test = ['pip']
```

Comment:
Why not drop the loop?

Reply:
Sorry, I should have dropped a comment here. The loop was kept as a reminder that this test is just a little bit in flux going forward. Checking just the version of pip is not ideal, but it's the only package that's both in our constraints and accessible from a setup.py. 

This test could be refactored to use CIBW_BEFORE_BUILD to do the asserts, but that would only give us the extra ability to check the version of delocate on macOS, nothing is preinstalled on windows. I didn't think it was worth it, especially because CIBW_DEPENDENCY_VERSIONS feels less useful than it once was because it no longer affects the actual build.

Reply:
I guess, it's possible that if we do get the ability to control build-system.requires dependencies some day, we might change CIBW_DEPENDENCY_VERSIONS to do that, and remove the ability to fiddle with the tool versions, or relegate it to a more 'under-the-hood' option.

Reply:
I see. May I suggest adding that as a code comment?

Reply:
reading this back now, there's a lot of what-ifs... i'll just remove the loop :)